ui/app/src/services/content.ts

```diff
@@ -398,8 +398,43 @@ function writeContentUrl(qs: object) {
   return `/studio/api/1/services/api/1/content/write-content.json?${qs.toString()}`;
 }
 
-export function updateField(modelId: string, fieldId: string, value: any): Observable<any> {
-  throw new Error('Not implemented.');
+export function updateField(site: string, modelId: string, fieldId: string, index: number, value: any): Observable<any> {
```

Comment:
Once you merge the latest updates, add the call to `updateModifiedDateElement(doc);`